CHANGES.rst

```diff
@@ -1,3 +1,27 @@
+3.0.0
+=====
+
+Summary
+-------
+
+Changes to Supported Platforms
+------------------------------
+
+- Starting with Falcon 3.0, Python 2 is no longer supported. Therefore,
+  support for CPython 2.7 and PyPy2.7 was removed in this release.
```

Comment:
Maybe we could add a recommendation to use Falcon 2.x in case Python 2.7 support is desired?